test/ssl-tests/01-simple.conf

```diff
@@ -11,6 +11,7 @@ ssl_conf = 0-default-ssl
 
 [0-default-ssl]
 server = 0-default-server
+server2 = 0-default-server2
```

Comment:
Hm, so with this approach, we still end up copying the entire context. I was originally thinking maybe we could do `server2 = 0-default-server`.

However, just leave it. It works fine and is not so important; I'll clean it up later if I can figure out how to.